### DISCUSSION TITLE
I need a mute command

### DISCUSSION THREAD

**Discussion by ghost** [Rapptz/discord.py#6327] [Q&A]
hi, I have no idea how to create a command to mute.  My problem (which I think is) I don't know how to collect the id of the user mentioned in the first argument, If a user mentioned is not found, go to the id of the user.  I leave an example:
```
if message.content.startswith("!mute")
    if message.author.guild_permissions.manage_roles:
        if mentioned_in(message.content[1]):
            memberBan = mentioned_user.id
             memberBan.ban(...)
...
```
The code is incomplete, but I need a sample code (that doesn't use discord.ext) to get my bearings. Since as you say I don't know how to separate the first argument that would be the command. See the id of the mentioned user, etc. Since most use Ext but I don't.

**Comment by 15696** [comment]
Why not use the commands extension? You can simplify all of this to:
```py
@commands.has_permissions(manage_roles = True)
@client.command()
async def mute(ctx, member: discord.Member):
    await member.ban(...)
```
(this will require a Bot instead of a Client)

**Reply by ghost** [reply]
Thanks, I was able to manipulate the arguments, of the message content. but discord.ext only interrupts what i am trying to do.

**Reply by 15696** [reply]
> Thanks, I was able to manipulate the arguments, of the message content. but discord.ext only interrupts what i am trying to do.

I doubt that. The commands extension is highly customizable, maybe you just didn't know what you were looking for. What were you trying to do?

**Comment by kapzy** [comment]
I highly recommend you learn python first

**Reply by ghost** [reply]
Don't worry, I'm now a pro at python-rust.

**Reply by bandwiches** [reply]
I'm sorry, but that is TOO funny.  HAHA!
